.circleci/docker/Dockerfile.clang.ubuntu1904

```diff
@@ -62,7 +62,7 @@ RUN git clone --recursive -b boost-1.69.0 https://github.com/boostorg/boost.git
     rm -rf /usr/src/boost
 
 # Z3
-RUN git clone --depth 1 -b z3-4.8.6 https://github.com/Z3Prover/z3.git \
+RUN git clone --depth 1 -b z3-4.8.7 https://github.com/Z3Prover/z3.git \
```

Comment:
@ekpyron how would it work to update it here though? Wouldn't this need the patch as well?

Reply:
Ah no, this is 64bit

Reply:
I hope at least, it should work, yes. If not we'll need to remove ``--depth 1`` and cherry-pick the commit here as well, but let's hope that's not necessary.